encode/Makefile.am

```diff
@@ -63,6 +63,13 @@ svctenc_LDADD		= \
 	$(top_builddir)/common/libva-display.la \
 	-lpthread -lm
 
+vp9enc_SOURCES     = vp9enc.c
+vp9enc_CFLAGS      = -I$(top_srcdir)/common -g
+vp9enc_LDADD       = \
+        $(LIBVA_LIBS) \
+    $(top_builddir)/common/libva-display.la \
+    -lpthread -lm
```

Comment:
Please fix the tab formatting here.  It should be single hard-tab.

Reply:
sure, thanks